### PR TITLE
5490 education history page

### PR DESCRIPTION
**Pull request by jbalboni**
Related to [713](https://github.com/department-of-veterans-affairs/vets.gov-team/issues/713)

The is the 5490 education history page, with the postHighSchoolTrainings piece shared with 1990e.

screen shot 2017-03-10 at 5 00 35 pm
screen shot 2017-03-10 at 5 00 18 pm
screen shot 2017-03-10 at 5 00 26 pm
screen shot 2017-03-10 at 5 00 05 pm
screen shot 2017-03-10 at 5 00 11 pm

### CODE REVIEW

File: src/js/edu-benefits/5490/config/form.js [hunk 1]
```diff
@@ -231,7 +239,128 @@ const formConfig = {
...
+          },
+          schema: {
+            type: 'object',
+            properties: {
+              highSchool: {
+                type: 'object',
+                properties: {
+                  status: highSchool.properties.status,
+                  highSchoolOrGedCompletionDate: date.schema,
+                  'view:hasHighSchool': {
```
Comment (cvalarida) [inline src/js/edu-benefits/5490/config/form.js]: Does this need to be flattened before sending to the api?

**Review by cvalarida — COMMENTED** [review]
So far so good. I just had one question.

File: src/js/edu-benefits/5490/config/form.js (hunk 1 above)
Comment (jbalboni) [inline src/js/edu-benefits/5490/config/form.js]: view: objects are automatically flattened

Comment (cvalarida) [inline src/js/edu-benefits/5490/config/form.js]: Good to know! I'll have to check on Monday to see if that's in the docs.

File: src/js/edu-benefits/utils/helpers.jsx [hunk 2]
```diff
@@ -157,3 +158,8 @@ export function showYesNo(field) {
 export function enumToNames(enumValues, names) {
   return enumValues.map(item => names[item]);
 }
+
+export const stateLabels = states.USA.reduce((current, { label, value }) => {
```
Comment (cvalarida) [inline src/js/edu-benefits/utils/helpers.jsx]: Could this be replaced with `_.mapKeys()` or `_.mapValues()`?

**Review by cvalarida — APPROVED** [review]
Other than making a wisecrack at
```js
import {
  benefitsLabels,
  relationshipLabels,
  highSchoolStatusLabels,
  transform
} from '../helpers';
```
I don't have much to say; it looks pretty good. I only had one suggestion for clarity.

File: src/js/edu-benefits/utils/helpers.jsx (hunk 2 above)
Comment (jbalboni) [inline src/js/edu-benefits/utils/helpers.jsx]: I don't think so. Those two transform object -> object, this is going from array -> object.

Comment (cvalarida) [inline src/js/edu-benefits/utils/helpers.jsx]: Ah, so it is. Alright.

**Pull request merged**